### ISSUE TITLE
Entity Relationship Diagrams show empty

### ISSUE THREAD

**Issue by lyj0309** [dbcodeio/public#153]
I have a pg db and have 400 count table, when i want get Entity Relationship Diagrams, the result is empty
![Image](https://github.com/user-attachments/assets/a6e72309-0bdc-423a-a6d2-b9d955ea720e)

**Comment by mikeburgh** [comment]
There must be something else going on other than the number of tables.. I created a simple script to create 400 related tables and was able to open it fine (scalled it up to 800 to test without any issues).. Not the most related of relations, but was a quick way to check size.

```sql
-- Create the primary table
CREATE TABLE primary_table (
    id SERIAL PRIMARY KEY,
    name VARCHAR(255)
);

-- Create 400 related tables
DO $$
DECLARE
    i INTEGER := 1;
BEGIN
    WHILE i <= 400 LOOP
        EXECUTE format('
            CREATE TABLE related_table_%s (
                id SERIAL PRIMARY KEY,
                primary_table_id INTEGER REFERENCES primary_table(id),
                data VARCHAR(255)
            );
        ', i);
        i := i + 1;
    END LOOP;
END $$;
```

Can you open up the developer tools (help -> Toggle developer tools, or command pallet: open webview developer tools) and share the errors/logs from the console, after you try and open the ERD ? It should hopefully give some indication of what failed.

If that has nothing helpful, would you mind sharing a script of your schema ? just the tables.. you can email it to me: mike@dbcode.io.

**Comment by lyj0309** [comment]
console:
```
webviewElement.ts:486 An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can escape its sandboxing.
diagram.js:1 Ignored call to 'alert()'. The document is sandboxed, and the 'allow-modals' keyword is not set.
diagram.js:2705 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'baseType')
    at diagram.js:2705:2403137
    at Array.map (<anonymous>)
    at zA.draw (diagram.js:2705:2403035)
    at zA.firstUpdated (diagram.js:2705:2400587)
    at zA._$AE (diagram.js:1:3589928)
    at zA.performUpdate (diagram.js:1:3589538)
    at zA.scheduleUpdate (diagram.js:1:3588246)
    at zA._$ET (diagram.js:1:3587948)
```

**Comment by mikeburgh** [comment]
Perfect, I can track that down and get it fixed

**Comment by mikeburgh** [comment]
Fixed in 1.5.8, I will close this, re open if it's still not working for you. Thanks for submitting it.
